app/components/decision_result_component.rb

```diff
@@ -22,7 +22,7 @@ def call
 
   def colour
     case result
-    when /fail/
+    when /fail|inel/
```

Comment:
I'm still not sure that inel will be sent by the MAAT API, but it could theoretically, so supporting it now.